Ambit/Source/Ambit/Mode/TestClasses/MockableGltfExporterExternal.h

```diff
@@ -0,0 +1,58 @@
+//   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License").
+//   You may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+#pragma once
+
+#include <utility>
+
+#include "Ambit/Mode/GltfExporterExternal.h"
+
+#include "MockableGltfExporterExternal.generated.h"
+
+/**
+ * Mock class for GltfExport.h
+ * To be used only during testing.
+ */
+UCLASS()
+class UGltfExporterExternalMock : public UObject, public IGltfExporterExternal
+{
+    GENERATED_BODY()
+public:
+    bool DoesExporterExist() override
+    {
+        return bExporterExists;
```

Comment:
There is a reason Mock classes don't have a corresponding cpp file:
1. Each function is a single line return. Creating a cpp file for these could potentially lead to file bloat as Ambit grows.
2. Once gMock is introduced, it will be reduced back to a single header file anyway